doc/apiary/v2/fiware-ngsiv2-rc-2018_04_30.apib

```diff
@@ -1841,7 +1841,7 @@ Response:
         ]
 
 
-### Create a new subscription [POST /v2/subscriptions]
+### Create Subscription [POST /v2/subscriptions]
 
 Creates a new subscription.
```

Comment:
Another decision to be made:
"Creates" or "Create"  ?
IMO, "Create" is better ...   (in the body text, I mean - the title is clearly 'Create')

Reply:
The description body should ideally be a single full sentence description of the endpoint. It doesn't need to be a copy of the title.  With *Swagger Code Gen* , the description body is copied to the generated documentation for the generated name


For Example : 

```
### Create Subscription [POST /v2/subscriptions]
 
 Creates a new subscription.
```
Would be displayed in a markdown table in the generated documentation as shown

```
| Name             | Endpoint               | Description                  |
|------------------|------------------------|------------------------------|
|createSubscription| POST /v2/subscriptions |  Creates a new subscription  |
```

Reply:
NTC or Fixed in [commit]?

(@jason-fox please have a look to the ["PR protocol"](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#pull-request-protocol) regarding how to end a comment thread before merging the PR. In this PR is not very important, as it is very simple, but better to get familiar with the procedure if you plan to contribute to the repo in a regularly basis ;). Thanks!)

Reply:
I would say NTC - The suggestion is to change/standardize the body of the text whereas the PR is focussed on the Titles only.